scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/DataSpaceServiceStarter.java

```diff
@@ -130,7 +132,7 @@ public static DataSpaceServiceStarter getDataSpaceServiceStarter() {
      */
     public void startNamingService() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)